client/my-sites/sidebar/sidebar.jsx

```diff
@@ -14,8 +14,7 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-var abtest = require( 'lib/abtest' ).abtest,
-	AdsUtils = require( 'lib/ads/utils' ),
+var AdsUtils = require( 'lib/ads/utils' ),
```

Comment:
This edit should be changed back; looks like `abtest()` is still used other places in the file (gives an `Uncaught ReferenceError: abtest is not defined` error).